instance.tf

```diff
@@ -0,0 +1,56 @@
+resource "aws_security_group" "web-sg" {
+  name   = "${var.env_code}-sg"
+  vpc_id = aws_vpc.main.id
+
+  ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["10.0.0.0/24", "10.0.1.0/24"]
```

Comment:
That's not your ISP address. ISP address is a public IP that is available via the internet. You can learn it from the website like https://whatismyipaddress.com/